docs/source/contrib_guide_docs.rst

```diff
@@ -72,4 +72,14 @@ Questions
 Feel free to ask questions in the Google Group for Jupyter or on an open issue
 on GitHub.
 
+=========================
+Git and Github Resources
+=========================
+
+If this is your first time working with Github or git, you can leverage the following
+resources to learn about the tools.
```

Comment:
pandas has a nice git guide for PRs and git rebase: https://github.com/pydata/pandas/wiki/Git-Workflows#git-rebase